geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/stats/RasterQueryStat.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2014 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.accumulo.stats
+
+import java.util.Date
+import java.util.Map.Entry
+
+import org.apache.accumulo.core.data.{Key, Mutation, Value}
+import org.calrissian.mango.types.encoders.lexi.LongReverseEncoder
+
+/**
+ * Class for capturing query-related stats
+ */
+case class RasterQueryStat(featureName:   String,
+                           date:          Long,
+                           rasterQuery:   String,
+                           planningTime:  Long,
+                           scanTime:      Long,
+                           mosaicTime:    Long,
+                           numResults:    Int) extends Stat
+
+/**
+ * Maps query stats to accumulo
+ */
+object RasterQueryStatTransform extends StatTransform[RasterQueryStat] {
+
+  private val CQ_QUERY = "rasterQuery"
+  private val CQ_PLANTIME = "timePlanning"
+  private val CQ_SCANTIME = "timeScanning"
+  private val CQ_MOSAICTIME = "timeMosaicing"
+  private val CQ_TIME = "timeTotal"
+  private val CQ_HITS = "hits"
+  val reverseEncoder = new LongReverseEncoder()
+  val NUMBER_OF_CQ_DATA_TYPES = 6
+
+  override def createMutation(stat: Stat) = {
+    new Mutation(s"${stat.featureName}~${reverseEncoder.encode(stat.date)}")
+  }
+
+  override def statToMutation(stat: RasterQueryStat): Mutation = {
+    val mutation = createMutation(stat)
+    val cf = createRandomColumnFamily
+    mutation.put(cf, CQ_QUERY, stat.rasterQuery)
+    mutation.put(cf, CQ_PLANTIME, stat.planningTime + "ms")
```

Comment:
Mightn't it be cleaner to add the unit designation to the CQ label?  That way, we could eliminate the "append 'ms'; remove 'ms' suffix" steps.